### ISSUE TITLE
Black failure on Travis-ci 

### ISSUE THREAD

**Issue by erwanp** [radis/radis#128]
### 🐛 Describe the bug

```
error: cannot format /home/travis/build/radis/radis/radis/levels/partfunc_cdsd.py: INTERNAL ERROR: Black produced code that is not equivalent to the source.  Please report a bug on https://github.com/psf/black/issues.  This diff might be helpful: /tmp/blk_xcyo54jw.log
```
Example : https://travis-ci.com/github/radis/radis/jobs/379490900#L266

But works perfectly with Local Black run (Python 3.6, Python 3.8) 

### 💡 Possible solutions

No idea at this point, but we may have to run Travis locally : 

https://stackoverflow.com/questions/21053657/how-to-run-travis-ci-locally

**Comment by erwanp** [comment]
Actually I could reproduce the bug using the Latest Black version (20.18)

https://github.com/psf/black/issues/1658

**Comment by erwanp** [comment]
Error was due to this change in Black : trailing commas are handled diffrently in 20.18 and 19.10. Different black version were used on the server and locally. Finally fixed in 7fcc0a772f7845c4b45566087e7e139f4f7ae2a7

https://github.com/psf/black/issues/1288
